### PR TITLE
[module] Add description

### PR DESCRIPTION
**Pull request by libmonsoon-dev**
## Description

Added module description

## Why do we need it, and what problem does it solve?

https://github.com/deckhouse/deckhouse/issues/5775

## What is the expected result?

When you add a description to module.yaml, it appears in module custom resource

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: deckhouse
type: feature
summary: Add module description.
impact_level: low
```

### CODE REVIEW

**Comment by libmonsoon-dev** [comment]
image

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @libmonsoon-dev. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @libmonsoon-dev. I've started the workflow for label 'e2e/run/static'...

File: modules/005-external-module-manager/module.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+description: "external module manager"
```
Comment (yalosev) [inline modules/005-external-module-manager/module.yaml:1]:
```suggestion
description: "Download and register modules from specified ModuleSources"
```

**Review by yalosev — APPROVED** [review]

**Review by nabokihms — APPROVED** [review]

**Pull request merged**